xAnalyzer/xanalyzer.cpp

```diff
@@ -328,14 +328,20 @@ void AnalyzeBytesRange(duint dwEntry, duint dwExit)
 			CallDestination = bii.addr;
 			DbgDisasmFastAt(CallDestination, &cbii);
 			GuiGetDisassembly(CurrentAddress, szDisasmText);
-			GuiGetDisassembly(bii.addr, szJmpDisasmText); // Detect function name on call scheme: CALL -> JMP -> JMP -> API
+			GuiGetDisassembly(CallDestination, szJmpDisasmText); // Detect function name on call scheme: CALL -> JMP -> JMP -> API
 
 			// save data for the argument
 			ai.manual = true;
 			ai.rvaEnd = CurrentAddress - Module::BaseFromAddr(CurrentAddress); // call address is the last		
 			if (Strip_x64dbg_calls(szDisasmText) || (cbii.branch && Strip_x64dbg_calls(szJmpDisasmText)))
 			{
 				szOriginalCharsetAPIFunction = szAPIFunction;
+
+				// Remove Stub suffix from function names if found
+				auto stub = szAPIFunction.find("Stub");
```

Comment:
Are you loading symbols or anything like that?, the thing is that nor x64dbg neither xAnalyzer plugin are responsible for that "Stub" suffix. Are you using any other plugins besides xAnalyzer?, Have you tried on a recent fresh x64dbg version with no plugins at all?. Merging this lines of code wouldn't suppose an issue, but I just wanna know the reason for this behaviour.